src/cmdlib.py

```diff
@@ -170,3 +170,52 @@ def get_basearch():
     except AttributeError:
         get_basearch.saved = RpmOstree.get_basearch()
         return get_basearch.saved
+
+
+class Builds:
+    def __init__(self, workdir=None):
+        self._workdir = workdir
+        self._fn = self._path("builds/builds.json")
+        if os.path.isfile(self._fn):
+            self._data = load_json(self._fn)
+        elif not os.path.isdir(self._path("builds")):
```

Comment:
Minor but seems like this could be moved up as a separate conditional - if there's no `builds/` there's definitely no `builds/builds.json`.

Reply:
Re-ordered this.